smos-query/package.yaml

```diff
@@ -37,6 +37,7 @@ library:
   - validity
   - validity-path
   - yaml
+  - ncurses
```

Comment:
Is this necessary? I'm surprised.

Reply:
Confirmed. I deleted this.  I was an attempt to fix the following error:
<img width="715" alt="image" src="https://github.com/NorfairKing/smos/assets/2463585/a058b540-75ea-477e-a3c5-dcb322a5d317">

However, it turned out the problem is that the curses library cannot be found on my windows machine. So for now, smos-query cannot be compiled.